### ISSUE TITLE
the sample image looks quite strange

### ISSUE THREAD

**Issue by PKUFlyingPig** [rosinality/style-based-gan-pytorch#92]
I run the generate.py on Mac which has cpu only, but the output sample image looks like this, what's the reason?
<img width="201" alt="image" src="https://user-images.githubusercontent.com/60659728/78692290-3ac91500-792c-11ea-9fda-3512909e6148.png">

**Comment by rosinality** [comment]
You need to set --size arguments which model is trained on.

**Comment by whjdarking** [comment]
Have you solve it yet? It seems I come across the same problem.

**Comment by PKUFlyingPig** [comment]
set --size arguments which model is trained on.

**Comment by whjdarking** [comment]
> set --size arguments which model is trained on.

Do you mean the size arguments in generate.py? I got a little confused

**Comment by PKUFlyingPig** [comment]
yes, the size depends on the resolution size of  image which your model trained on.

**Comment by whjdarking** [comment]
> yes, the size depends on the resolution size of image which your model trained on.

Thank you!
